src/sqlalchemy_cratedb/dialect.py

```diff
@@ -258,6 +267,8 @@ def get_schema_names(self, connection, **kw):
 
     @reflection.cache
     def get_table_names(self, connection, schema=None, **kw):
+        if schema is None:
```

Comment:
nitpick: It could be just
 ```python
 if not schema:
```

Reply:
Thanks. Sometimes, objects don't implement the `__bool__` protocol well, so I made it a habit to favor "explicit over implicit", also following the recommendation of Python Zen.